src/main/java/jp/co/future/uroborosql/SqlAgentImpl.java

```diff
@@ -524,6 +524,35 @@ public SqlUpdate updateWith(final String sql) {
 		return new SqlUpdateImpl(this, context().setSql(sql));
 	}
 
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @see jp.co.future.uroborosql.SqlAgent#updateChained(java.lang.String[])
+	 */
+	@Override
+	public SqlUpdate updateChained(final String... sqlNames) {
+		if (sqlNames == null || sqlNames.length == 0) {
+			throw new IllegalArgumentException("sqlNames is required.");
+		}
+		if (sqlNames.length == 1) {
+			warnWith(LOG)
+					.setMessage("If sqlNames is single, use update method instead of updateChained.")
+					.log();
+			return update(sqlNames[0]);
+		}
+		if (!getDialect().supportsUpdateChained()) {
+			throw new UroborosqlRuntimeException(getDialect().getDatabaseName() + " does not support updateChained.");
+		}
+		var sqls = Arrays.stream(sqlNames)
+				.map(sqlName -> getSqlResourceManager().getSql(sqlName))
+				.collect(Collectors.joining(";" + System.lineSeparator()));
+
+		var sqlName = Arrays.stream(sqlNames)
+				.collect(Collectors.joining(","));
```

Comment:
@HidekiSugimoto189 
If you just want to convert to a comma-separated string, I think the following would be smarter.
```suggestion
		var sqlName = String.join(",", sqlNames));
```

Reply:
Thanks review. fixed it.